pkg/component/etcd/testdata/monitoring_alertingrules_important_multinode_with_backup.yaml

```diff
@@ -100,31 +100,23 @@ tests:
     alertname: KubeEtcdDeltaBackupFailed
     exp_alerts:
     - exp_labels:
-        pod: etcd
-        role: test
-        job: kube-etcd3-backup-restore-main
-        kind: Incr
```

Comment:
Is losing these labels (see above) intended? I think we would no longer be able to distinguish etcd-main and etcd-events alerts.

Reply:
It was failing some test cases. The labels were only for some test cases. But in the new update, I reintroduced this labels in the testcase file and tackled the problem by adding the labels in main alert string as.